articles/azure-monitor/snapshot-debugger/snapshot-debugger.md

```diff
@@ -28,11 +28,9 @@ You can view debug snapshots in the portal to see the call stack and inspect var
 
 Snapshot collection is available for:
 
-- .NET Framework and ASP.NET applications running .NET Framework 4.6.2 and newer versions.
-- .NET and ASP.NET applications running .NET [LTS](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) and newer versions on Windows.
-- .NET [LTS](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) (and newer versions) applications on Windows.
+- .NET Framework 4.6.2 and newer versions.
+- .NET 6+ on Windows.
```

Comment:
```suggestion
- .NET 6.0 or later on Windows.
```
not sure of our style but I Think it should have the `.0`

Reply:
we could gratuitiously link to the .NET Core download site here?

Reply:
```suggestion
- [.NET 6 or later](https://dotnet.microsoft.com/download) on Windows.
```